app.py

```diff
@@ -83,6 +83,17 @@ def get_harvest_error(error_id):
     except Exception:
         return "provide error_id"
 
+
+@app.route("/harvests/<string:source_id>", methods=["PUT"])
+def update_harvest_source():
+    source_id = request.args.get("source_id", None)
```

Comment:
`source_id` is available as a function argument, why it is getting from querystring here?

Reply:
on a different note I like the use of variables in the route itself like you have here instead of parsing them via `requests.get( "example" )` from the query string. i think it's more descriptive of what's expected vs just `/harvests`

Reply:
Sorry, good catch. That is how it was in the `harvesting-orchestrator` so I just copied.